<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanFrameComposerTab</class>
 <widget class="QWidget" name="CanFrameComposerTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="selectInterfaceBox"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolBar" name="toolBar"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Interval (ms):  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sendInterval">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkLoopEnabled">
       <property name="text">
        <string>Loop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
        </font>
       </property>
       <property name="text">
        <string>0/0 Frames | Time remaining: 00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="composerText">
     <property name="font">
      <font>
       <family>Source Code Pro</family>
      </font>
     </property>
     <property name="plainText">
      <string>111 FF 88 11 22</string>
     </property>
     <property name="placeholderText">
      <string>4F8 03 FF 80 AC DD</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
